frontend/junior-1/html.md

```diff
@@ -50,18 +50,21 @@
         </ul>
     </header>
     ```
-* Зачем нужны метатеги?
-* Что такое data-атрибуты, и привести примеры, когда они полезны?
-* Когда стоит использовать тег `img`, а когда `background-image`?
-* Как стилизовать `svg`? Какие основные атрибуты и CSS-свойства используются для этого?
-* Что такое специальные html-сущности (неразрывный пробел, тире, которое не дефис) и как с ними работать?
+* What are meta tags for?
+* What are data attributes, and give examples of when they are useful?
+* When should the `img` tag be used, and when should `background-image` be used?
+* How to style `svg`? What are the main attributes and CSS properties used for this?
+* What are special HTML entities (non-breaking space, dash, which is not a hyphen) and how to work with them?
 
-### Ресурсы
+### Resources
 
-* [Людоедский интерфейс, Вадим Макеев](https://www.youtube.com/watch?v=ssJsjGZE2sc)
-* [HTML Academy](https://htmlacademy.ru/)
+* [Ludoeedskiy interfeys, Vadim Makeev](https://www.youtube.com/watch?v=ssJsjGZE2sc)
```

Comment:
Can be replaced with [this article](https://www.htmhell.dev/adventcalendar/2023/18/)